doc/man1/openssl-pkcs12.pod.in

```diff
@@ -166,6 +167,16 @@ always assumes these are the same so this option will render such
 PKCS#12 files unreadable. Cannot be used in combination with the options
 B<-password>, B<-passin> if importing, or B<-passout> if exporting.
 
+=item B<-legacy>
+
+This option is used to fallback to the default algorithms as in the legacy
+provider. In the legacy mode, the default algorithm for certificate encryption
```

Comment:
I would change the first sentence to something like: `Use legacy mode of operation and automatically load the legacy provider.`